includes/class-sensei-admin.php

```diff
@@ -1426,24 +1426,28 @@ public function handle_order_lessons() {
 			wp_die( esc_html__( 'Insufficient permissions', 'sensei-lms' ) );
 		}
 
-		$ordered = null;
+		$course_id = (int) $_POST['course_id'] ?? null;
```

Comment:
Did you mean?
```
$course_id = isset($_POST['course_id'] ? (int) $_POST['course_id'] : null
```

In the current state, the interpreter will execute this code as:
```
$course_id = (int) ($_POST['course_id'] ?? null);
```

Here is how it looks in the interactive shell (PHP 7.4):
```
% php -a
Interactive shell

php > $a = ['x' => '1'];
php > $x = (int) $a['x'] ?? null;
php > var_dump($x);
php shell code:1:
int(1)
php > $y = (int) $a['y'] ?? null;
PHP Notice:  Undefined index: y in php shell code on line 1
PHP Stack trace:
PHP   1. {main}() php shell code:0

Notice: Undefined index: y in php shell code on line 1

Call Stack:
   96.2171     393120   1. {main}() php shell code:0

php > var_dump($y);
php shell code:1:
int(0)
```

I think that's not critical but might lead to some misunderstanding in the future expecting `null` when `course_id` was not provided.

Reply:
You are totally right. Thanks for catching this. Fixed in fd689d643f7df4f40ff237e096cae8f3407b19f4.